app/views/admin/creator_settings/_form.html.erb

```diff
@@ -16,7 +16,7 @@
     <p id="color-selector-subtitle" class="crayons-field__description">This will be the "accent" color used for buttons, links, etc.</p>
   <% end %>
 
-  <div class="flex w-100 m:w-50 crayons-field">
+  <div class="flex w-100 m:w-100 crayons-field">
```

Comment:
Refactor (blocking): No need for width CSS utility classes as the `flex` CSS utility class will take the entire space available and adjust as the page resizes. Also, the `.crayons-field` already sets the `display` to `flex`.

![image](https://user-images.githubusercontent.com/833231/145596039-1460a740-8887-458f-bc55-52cf02fabe5a.png)

```suggestion
  <div class="crayons-field">
```